client/src/components/CreateRoseBudThorn.jsx

```diff
@@ -0,0 +1,13 @@
+
+export default function CreateRoseBudThorn() {
+  return (
+    <div classname="create" >
```

Comment:
```suggestion
    <div className="create" >
```

Reply:
Another package suggestion eslint 
https://www.npmjs.com/package/eslint

It will warn you of this kind of typos.

Reply:
people also pair eslint with prettier to make everything pretty
https://www.npmjs.com/package/prettier

Reply:
thank you both!  i tried adding prettier and also researched the web, but could not get it to work.  I'll keep working on it!
I changed the typo.